### ISSUE TITLE
Option to disable the 'Share' and 'Lookup' action on Text widgets

### ISSUE THREAD

**Issue by evolanakis** [eclipsesource/tabris#448]
On iOs, when selecting some text in a Text widget, the following actions are now available:
- Lookup (web search)
- Share (i.e. send to other app or device)

For sensitive apps, it is desirable to disable those actions.

New API will be added to TextDecorator:

```
TextDecorator.disableLookupAction();
TextDecorator.disableShareAction();
```

**Comment by evolanakis** [comment]
Please review commits for #447 in our gerrit (1x java, 1x iOS)

**Comment by patrykmol** [comment]
Done.

**Comment by evolanakis** [comment]
Available in 1.5-maintenance
